cdap-ui/app/features/tracker/controllers/integrations-ctrl.js

```diff
@@ -63,16 +83,28 @@ class TrackerIntegrationsController {
 
     this.pollId = null;
 
+    $scope.$watch(() => {
```

Comment:
```
$scope.$watch('TrackerIntegrationsController.navigatorSetup.isOpen', () => {...});
```
Is it possible to use just the controller variable instead of function callback?